### ISSUE TITLE
Birdseye miscalculates location of landscape cameras in row after portrait camera

### ISSUE THREAD

**Issue by ccutrer** [blakeblackshear/frigate#7062]
### Describe the problem you are having

In this screenshot, the camera in the third row and the third column is portrait. It's properly sizing it as maximum double-row height. BUT... the cameras on the fourth row are covering the bottom half of it. Oddly, these cameras have some padding on the left that are the exact width of the portrait camera on row 3 - almost as if they're _trying_ to make room for it, but didn't account for the portrait camera's x-offset in the prior row.

<img width="1043" alt="Screenshot 2023-07-06 at 2 23 34 PM" src="https://github.com/blakeblackshear/frigate/assets/191320/ae266a49-66e6-48ff-a178-9f99ee1ca3db">

This happens anytime the portrait camera is active in birdseye, and there are enough other cameras active _after_ the portrait one to need another row - that row is offset the width of the portrait camera. on the left, and covers the portrait camera wherever it is (unless it's the first camera in the prior row, then it would work out? I haven't yet seen that state though, since my portrait camera is pretty far down the list alphabetically).

### Version

0.13.0-DD02958

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

M.2

### Network connection

Wired

### Camera make and model

N/A

### Any other information that may be helpful

Birdseye is restreamed, at 1920x1080. All detect streams are 1280x720 except the top right in this screenshot (doorbell) is 960x720, and the portrait camera (south_side) is 360x640.

**Comment by NickM-27** [comment]
Yeah I think I had done that for some reason assuming the portrait camera was on the left... going to be a complicated problem to solve especially in the case of multiple portrait cameras

**Comment by cefoster0** [comment]
Bumping this issue. It seems like since 15.1, this is occurring more and more often.

**Comment by grodcur22** [comment]
Any updates on this?
